src/components/LoanCards/KivaClassicBasicLoanCard.vue

```diff
@@ -65,7 +65,9 @@
 		<borrower-name
 			v-if="!isLoading"
 			class="tw-mb-1 tw-text-h3"
+			:max-length="50"
 			:name="borrowerName"
+			style="min-height: 60px;"
```

Comment:
Text h3 has a line-height of 30 so we double that to get a baseline height. min-height allows the container to scale if someone zooms their screen...